### PR TITLE
Add event endpoints on find and get

### PR DESCRIPTION
**Pull request by VLiamin**

### CODE REVIEW

File: src/EventService.Broker/Requests/ImageService.cs [hunk 1]
```diff
@@ -46,5 +54,18 @@ public async Task<List<Guid>> CreateImagesAsync(List<ImageContent> eventImages,
...
+  {
+    return imagesIds is null || !imagesIds.Any()
+      ? default
+      : (await RequestHandler.ProcessRequest<IGetImagesRequest, IGetImagesResponse>(
+          _rcGetImages,
+          IGetImagesRequest.CreateObj(imagesIds, ImageSource.Event),
+          errors,
+          _logger))
+        ?.ImagesData
+        .Select(_mapper.Map).ToList();
```
Comment (AnnPostnikova) [inline src/EventService.Broker/Requests/ImageService.cs:69]: ConvertAll()

Comment (AnnPostnikova) [inline src/EventService.Broker/Requests/ImageService.cs]: Better ConvertAll()

File: src/EventService.Business/Commands/Event/FindEventsCommand.cs [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    IEventInfoMapper mapper)
+  {
+    _repository = repository;
+    _mapper = mapper;
+  }
+
+  public async Task<FindResultResponse<EventInfo>> ExecuteAsync(FindEventsFilter filter, CancellationToken ct = default)
+  {
+    (List<DbEvent> events, int totalCount) =
+      await _repository.FindAsync(filter: filter, ct: ct);
```
Comment (SolovyovDmitry) [inline src/EventService.Business/Commands/Event/FindEventsCommand.cs]: parameters are so obvious so I'm not sure it's required

File: src/EventService.Business/Commands/Event/FindEventsCommand.cs [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+    if (events is null || !events.Any())
+    {
+      return new();
```
Comment (SolovyovDmitry) [inline src/EventService.Business/Commands/Event/FindEventsCommand.cs:56]: won't it be the same as default?

File: src/EventService.Business/Commands/Event/GetEventCommand.cs [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+
+    List<FileCharacteristicsData> files = await filesTask;
+    List<UserData> users = await usersTask;
+    List<ImageInfo> images = await imagesTask;
+
+    List<CommentInfo> comments = null;
+
+    if (filter.IncludeComments)
+    {
+      comments = dbEvent.Comments.Select(_commentInfoMapper.Map).ToList();
```
Comment (SolovyovDmitry) [inline src/EventService.Business/Commands/Event/GetEventCommand.cs:78]: ConvertAll ?

**Review by SolovyovDmitry — APPROVED** [review]

File: src/EventService.Mappers/Models/EventInfoMapper.cs [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+  public EventInfo Map(DbEvent dbEvent)
+  {
+    return dbEvent is null
+      ? null
+      : new EventInfo
+      {
+        Id = dbEvent.Id,
+        Name = dbEvent.Name,
+        Description = dbEvent.Description,
+        Date = dbEvent.Date,
```
Comment (AnnPostnikova) [inline src/EventService.Mappers/Models/EventInfoMapper.cs:26]: maybe add EndDate?

File: src/EventService.Models.Dto/Requests/Event/FindEventsFilter.cs [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+  [FromQuery(Name = "endTime")]
+  public DateTime? EndTime { get; set; }
+
+  [FromQuery(Name = "access")]
+  public AccessType? Access { get; set; }
+
+  [FromQuery(Name = "color")]
+  public CategoryColor? Color { get; set; }
+}
```
Comment (AnnPostnikova) [inline src/EventService.Models.Dto/Requests/Event/FindEventsFilter.cs:33]: maybe add IncludeDeactivated and Category name?

**Review by AnnPostnikova — APPROVED** [review]

File: src/EventService.Business/Commands/Event/FindEventsCommand.cs (hunk 3 above)
Comment (ndoriomedov) [reply src/EventService.Business/Commands/Event/FindEventsCommand.cs:56]: no, default for reference types is null

**Review by ndoriomedov — APPROVED** [review]

File: src/EventService.Business/Commands/Event/GetEventCommand.cs (hunk 4 above)
Comment (VLiamin) [reply src/EventService.Business/Commands/Event/GetEventCommand.cs:78]: ICollection does not have this method

**Pull request merged**